src/Integration/ProfileConflicts/ConflictsManager.cs

```diff
@@ -150,7 +150,7 @@ private IReadOnlyList<ProjectRuleSetConflict> FindConflicts(RuleSetInformation[]
                     continue;
                 }
 
-                if (projectBinderFactory.Get(project) == null)
+                if (projectBinderFactory.Get(project) is CFamilyProjectBinder)
```

Comment:
I'm really not a fan of this, but this should be temporary:  the logic should eventually move to ProjectBinder.

Reply:
The intent would have been clearer if the check have been against `RoslynProjectBinder` (i.e. "we only want to continue if the binder is of type X"), but it doesn't matter if it's temporary.